easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'MMseqs2'
+version = '11-e1a1c'
+
+homepage = 'https://mmseqs.com'
+description = "MMseqs2: ultra fast and sensitive search and clustering suite"
+
+toolchain = {'name': 'iimpi', 'version': '2019b'}
+
+source_urls = ['https://github.com/soedinglab/MMseqs2/archive/']
```

I would suggest to better use `GITHUB_SOURCE` for simplicity
```suggestion
github_account = 'soedinglab'
source_urls = [GITHUB_SOURCE]
```